Library/Homebrew/os/mac/version.rb

```diff
@@ -30,6 +30,20 @@ def self.from_symbol(version)
         new(str)
       end
 
+      def outdated_release?
+        # TODO: bump version when new macOS is released and also update
+        # references in docs/Installation.md and
+        # https://github.com/Homebrew/install/blob/HEAD/install.sh
+        version < "10.14"
```

Comment:
I'd like to propose to keep changeable parameters at the top of the class by defining couple of constants right after `SYMBOLS` (e.g., `oldest_supported` and `newest_supported`, or anything along these lines) and use these constants in these two methods.

Reply:
That's a good solution to this issue. I'll add it in.